module-4/backend/week_four.md

```diff
@@ -4,16 +4,28 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-1. What's your favorite project management tool?
-2. Why do we create staging environments?
-3. What are the characteristics of a good README (in your opinion)?
+1. What's your favorite project management tool?  
+* Waffle
+2. Why do we create staging environments?  
+* To try to emulate the production environment as closely as possible and run tests before deploying
+3. What are the characteristics of a good README (in your opinion)?  
+* Clear, full sentences
+* Exact command line instructions for unusual situations
 4. What's one main improvement you're going to make to your code regarding accessibility issues?
+* Making sure every button and link can be accessed by tab/enter
 5. What are some basic security concerns to be aware of when building applications?
+* SQL injection, privilege escalation
 6. Why is continuous integration helpful/important?
+* It makes sure your new changes don't break your old code
 7. What are some continuous integration tools?
+* TravisCI, Jenkins
 
 #### Review  
 
 8. What are some characteristics of "good" git workflow?
+* Using branches for features, not merging your own pull requests
 9. What are the four fundamental concepts of object oriented programming?
+* Abstraction, encapsulation, inheritance, polymorphism
 10. What's a module in Ruby and what's the difference between a class and a module?
+* A module is a way to provide common methods to two similar but distinct classes
```

Comment:
how is this different from using a class to provide common methods to two similar but distinct classes? what are some use cases of modules?